hack/charts/mysql-operator/templates/podsecuritypolicy.yaml

```diff
@@ -0,0 +1,53 @@
+{{- if .Values.podSecurityPolicy }}
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: mysql-operator
+  annotations:
```

Comment:
I think annotations should be changable. Maybe define a `.Values.podSecurityPolicy.annotations` like some charts in https://github.com/helm/charts do.

Reply:
done